api/controllers/TelevisionController.js

```diff
@@ -0,0 +1,328 @@
+     */
+    rewind: function(req, res, next){
+        gladys.television.rewind(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/pause Pause 
+     * @apiName televisionPause
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    pause: function(req, res, next){
+        gladys.television.pause(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/play Play 
+     * @apiName televisionPlay
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    play: function(req, res, next){
+        gladys.television.play(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/muted Set muted 
+     * @apiName televisionSetMuted
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * @apiParam {boolean} muted true if sound is muted
+     */
+    setMuted: function(req, res, next){
+        gladys.television.setMuted(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+     /**
+     * @api {post} /television/volume Set Volume 
+     * @apiName televisionSetVolume
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * @apiParam {integer} volume Volume level
+     */
+    setVolume: function(req, res, next){
+        gladys.television.setVolume(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/stop Stop 
+     * @apiName televisionStop
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    stop: function(req, res, next){
+        gladys.television.stop(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/volumeUp Volume Up 
+     * @apiName televisionVolumeUp
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    volumeUp: function(req, res, next){
+        gladys.television.volumeUp(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/volumeDown Volume Down 
+     * @apiName televisionVolumeDown
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    volumeDown: function(req, res, next){
+        gladys.television.volumeDown(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/pressKey Press Key 
```

Comment:
Same here

Reply:
Fix this on commit c62f1694228a420b53efebcb015d877208a3c3d3